setup.py

```diff
@@ -451,6 +451,10 @@
                     'proteus/tests/hex_cube_3x3.h5']),
                   (os.path.join(proteus_install_path,'tests','linalgebra_tests'),
                    ['proteus/tests/linalgebra_tests/sparse_mat_1.txt']),
+                  (os.path.join(proteus_install_path,'tests','solver_tests','import_modules'),
+                   ['proteus/tests/solver_tests/import_modules/quad_mass_matrix.npy',
+                    'proteus/tests/solver_tests/import_modules/sol_10.npy',
+                    'proteus/tests/solver_tests/import_modules/sol_20_lst.npy']),
```

Comment:
in the future we want to track these binary files with git lfs instead of git. @zhang-alvin has done this fairly extensively.